src/calendar-heatmap.js

```diff
@@ -62,14 +91,40 @@ function calendarHeatmap() {
     return chart;
   };
 
-  function chart() {
+  chart.onMouseOver = function (value) {
+    if (!arguments.length) { return onMouseOver(); }
+    onMouseOver = value;
+    return chart;
+  };
 
+  chart.onMouseOut = function (value) {
+    if (!arguments.length) { return onMouseOut(); }
```

Comment:
oh actually if u look at the lines before this, the author wrote 
`if (!arguments.length) { return onClick(); }`

I think the intention is to trigger the event.